scripts/apps/authoring/views/send-item.html

```diff
@@ -101,6 +101,14 @@
                     {{!canSendItem() ? 'fetch' : 'send' | translate}}
                 </button>
             </li>
+            <li class="full-width" ng-if="showSendButtonAndDestination() && isPackage">
+                <button class="btn btn--large btn--primary btn--expanded"
+                        ng-click="send(false, true)"
+                        ng-disabled="disableSendButton()"
+                        >
+                    {{'send all' | translate}}
```

Comment:
Use `<span translate>send all</span>`. When translating using the filter, strings don't get registered for translation by grunt task.